asserts/pool.go

```diff
@@ -170,6 +200,53 @@ func (u *unresolvedRec) merge(at *AtRevision, gnum uint16, gr *internal.Grouping
 	}
 }
 
+type unresolvedSeqRec struct {
+	at       *AtSequence
+	grouping internal.Grouping
+
+	serializedLabel Grouping
+
+	err error
+}
+
+func (u *unresolvedSeqRec) groupingPtr() *internal.Grouping {
+	return &u.grouping
+}
+
+func (u *unresolvedSeqRec) label() Grouping {
+	return u.serializedLabel
+}
+
+func (u *unresolvedSeqRec) isAssertionNewer(a Assertion) bool {
+	return a.Revision() > u.at.Revision
+}
+
+func (u *unresolvedSeqRec) revision() int {
+	return u.at.Revision
+}
+
+func (u *unresolvedSeqRec) error() error {
+	return u.err
+}
+
+func (u *unresolvedSeqRec) exportTo(r map[Grouping][]*AtSequence, gr *internal.Groupings) {
+	serLabel := Grouping(gr.Serialize(&u.grouping))
+	// remember serialized label
+	u.serializedLabel = serLabel
+	r[serLabel] = append(r[serLabel], u.at)
+}
+
+func (u *unresolvedSeqRec) merge(at *AtSequence, gnum uint16, gr *internal.Groupings) {
+	gr.AddTo(&u.grouping, gnum)
+	// use highest revision & sequence
+	if at.Revision > u.at.Revision {
```

Comment:
this check as is makes sense only if at and u.at have the same sequence.  Sequence and revision are not independent of each other,  revision range is different for each sequence point, as each sequence point is a different assertion